src/components/20-molecules/dropdown/react/DemoUncontrolledDropdownReact.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
Maybe we need demo that the onFocus and onBlur Props Callbacks work in react environment.

Reply:
done